src/update-handling/FileService.ts

```diff
@@ -68,29 +69,39 @@ export class FileService {
                     }
                 }
 
-                if (l.startsWith("Hotkey=")) {
+                if (lowerCaseLine.startsWith("hotkey=")) {
                     currentHotkey.hotKey = this.extractRightString(l);
                 }
 
-                if (l.startsWith("Unhotkey=")) {
+                if (lowerCaseLine.startsWith("unhotkey=")) {
                     currentHotkey.unHotkey = this.extractRightString(l);
                 }
 
-                if (l.startsWith("Researchhotkey=")) {
+                if (lowerCaseLine.startsWith("researchtip=")) {
+                    currentHotkey.researchTip = this.extractRightString(l);
+                }
+
+                if (lowerCaseLine.startsWith("researchhotkey=")) {
                     currentHotkey.researchHotkey = this.extractRightString(l);
                 }
 
-                if (l.startsWith("Buttonpos=")) {
+                if (lowerCaseLine.startsWith("tip=")) {
+                    currentHotkey.tip = this.extractRightString(l);
+                }
+
+                if (lowerCaseLine.startsWith("untip=")) {
+                    currentHotkey.untip = this.extractRightString(l);
+                }
+
+                if (lowerCaseLine.startsWith("buttonpos=")) {
                     const positions = l.split("=")[1].split(",");
                     if (positions.length === 2) {
                         currentHotkey.grid = new Grid(parseInt(positions[0]), parseInt(positions[1]))
                     }
                 }
             })
 
-            if (currentHotkey.hotKey && currentHotkey.hotkeyCommand) {
-                hotkeys.push(currentHotkey)
-            }
+            hotkeys.push(currentHotkey)
```

Comment:
that will push a lot of hotkeys that are not parse correctly

Reply:
🤔 that's probably a mistake on my end. I'll double check!